dicom_utils/cli/project.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import argparse
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Union
+
+import numpy as np
+import pydicom
+
+from ..dicom import convert_frame_voi_lut, decompress
+
+
+def get_parser(parser: ArgumentParser = ArgumentParser()) -> ArgumentParser:
+    parser.add_argument("path", type=Path, help="DICOM file to decompress")
+    parser.add_argument("dest", type=Path, help="destination DICOM filepath")
+    parser.add_argument("-m", "--method", default="max", choices=["max", "mean", "middle"], help="method of reduction")
+    parser.add_argument(
+        "-g", "--gpu", default=False, action="store_true", help="use NVJPEG2K accelerated decompression"
+    )
+    parser.add_argument(
+        "-b", "--batch-size", default=4, type=int, help="batch size for NVJPEG2K accelerated decompression"
```

Comment:
Default to `None` to respect env var `NVJPEG2K_BATCH_SIZE`
```suggestion
        "-b", "--batch-size", default=None, type=int, help="batch size for NVJPEG2K accelerated decompression"
```